README.md

```diff
@@ -197,6 +198,17 @@ jwt.verify(token, cert, { algorithms: ['RS256'] }, function (err, payload) {
   // if token alg != RS256,  err == invalid signature
 });
 
+// Verify using getKey callback
+function getKey(header, callback) {
+  // fetch secret or public key
```

Comment:
I'd add some comment encouraging the users to cache the key with the _kid_ so they avoid fetching it multiple times.

Reply:
I will add https://github.com/auth0/node-jwks-rsa as an example to load the keys, that should make it more clear.